### PR TITLE
CNV-32614: Document deprecate templates functionality

### PR DESCRIPTION
**Pull request by aspauldi**
Version(s): 4.14+

Issue: [CNV-32614](https://issues.redhat.com/browse/CNV-32614)

Links to docs previews:
- [About VM templates (Customization)](https://72894--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/virtual_machines/creating_vms_rh/virt-creating-vms-from-templates#virt-about-templates)
- [Customizing a VM template](https://72894--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/virtual_machines/creating_vms_rh/virt-creating-vms-from-templates#virt-customizing-vm-template-web_virt-creating-vms-from-templates)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by aspauldi** [comment]
Hi @ksimon1 , can you please review this doc update? 

Two comments:

- The reorganization of the web console documentation in 4.14 has removed the 4.12 (and 4.13) version of the topic that was originally tagged as being incorrect in the Jira. Therefore, this doc update is being applied only to 4.14+.

- I'm not able to see this situation in the cluster I was using, so please review the procedure carefully. I made some assumptions. 

Thanks.

**Comment by aspauldi** [comment]
@ksimon1, if you have a chance, can you take a look at this doc update? (I'm going to be out of the office from March 13 through March 29.) Thanks!

**Comment by aspauldi** [comment]
@geetikakay, can you please review this PR? Thanks!

**Comment by aspauldi** [comment]
@geetikakay, can you please review this PR? Thanks! (I was on PTO for two weeks but am back now.)

**Comment by aspauldi** [comment]
@geetikakay, can you please review this PR? Thanks!

**Comment by geetikakay** [comment]
lgtm

**Review by skopacz1 — COMMENTED** [review]
Just a few comments but otherwise LGTM!

File: modules/virt-customizing-vm-template-web.adoc [hunk 1]
```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/creating_vms_rh/virt-creating-vms-from-templates.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-customizing-vm-template-web_{context}"]
+= Customizing a VM template by using the web console
+
+You can customize an existing VM template by modifying the VM or template parameters, such as data sources, cloud-init, or SSH keys, before you start the VM. If you customize a template by copying it and including all of its labels and annotations, the customized template is marked as deprecated when a new version of the Scheduling, Scale, and Performance (SSP) Operator is deployed.
```
[the remark this answers is not in the document]
Comment (aspauldi) [reply modules/virt-customizing-vm-template-web.adoc:9]: The related topics in this section all use VM in the headings, so I used the same to keep it consistent. I don't want to change one without changing them all, which exceeds the scope of this PR, I think.
CNV-32614

File: virt/virtual_machines/creating_vms_rh/virt-creating-vms-from-templates.adoc [hunk 2]
```diff
@@ -23,6 +23,11 @@ You can customize the disk source and VM parameters before you start the VM:
 
 * See the xref:../../../virt/getting_started/virt-web-console-overview.adoc#virtualmachine-details-overview_virt-web-console-overview[*Overview*], xref:../../../virt/getting_started/virt-web-console-overview.adoc#virtualmachine-details-yaml_virt-web-console-overview[*YAML*], and xref:../../../virt/getting_started/virt-web-console-overview.adoc#virtualmachine-details-configuration_virt-web-console-overview[*Configuration*] tab documentation for details about VM settings.
 
+[NOTE]
+====
+If you copy a VM template with all its labels and annotations, your version of the template is marked as deprecated when a new version of the Scheduling, Scale, and Performance (SSP) Operator is deployed. You can remove this designation. See xref:../../virt/virtual_machines/creating_vms_rh/virt-customizing-vm-template-web.adoc#virt-customizing-vm-template-web_virt-creating-vms-from-templates[Customizing a VM template].
```
[the remark this answers is not in the document]
Comment (aspauldi) [reply virt/virtual_machines/creating_vms_rh/virt-creating-vms-from-templates.adoc:28]: Thanks, Sebastian!

**Comment by skopacz1** [comment]
Hey @aspauldi I'm proceeding with the merge review, but I just have a question - do you know if this needs change management? If this is the first time the feature is being mentioned in published docs (i.e. if it wasn't mentioned in a release note when the functionality first became GA), I feel like it might, but I'm not sure. Let me know what you think!

**Comment by aspauldi** [comment]
Hi Sebastian, we don't do change management in CNV. Thanks for checking!

**Review by skopacz1 — APPROVED** [review]
Merge review LGTM!

**Pull request merged**